concepts/Common/DevExtreme CLI/10 Add DevExtreme to an Existing Application.md

```diff
@@ -43,6 +43,8 @@ This command does the following:
 
 These steps are explained in the [Add DevExtreme to a React Application](/concepts/50%20React%20Components/05%20Add%20DevExtreme%20to%20a%20React%20Application/00%20Add%20DevExtreme%20to%20a%20React%20Application.md '/Documentation/Guide/React_Components/Add_DevExtreme_to_a_React_Application/') article.
 
+The `add devextreme-react` command supports React apps created with various build tools, such as Vite and Next.js. DevExtreme CLI supports Next.js apps created with different option combinations, such as with or without a `src` folder, using TypeScript or JavaScript, with or without app routing, and more.
```

Comment:
"created with various build tools" sounds like other (except vite and nextjs) are also supported.
may be better to write smth like "created by create-vite or next.js" ?
"created with different option combinations" - also very wide. 
add devextreme-react supports apps created by nextjs with combinations of options:
- use TypeScript or not
- code inside a `src/` or not
- use App Router or not

Project structure depends on all this options and "add devextreme-react" supports all this combinations.